docs/getting-started/quick-start.md

```diff
@@ -4,12 +4,330 @@ sidebar_position: 2
 
 # Quick Start
 
-In this section we are providing a sample application that one can deploy and just use to play around.
+In this quik start, you will see a sample application that you deploy and use to
+play around.  As part of this exercise, a backing service and an application is
+required.  The backing service is a PostgreSQL database and application is a
+[Spring Boot REST API server][petclinic].
 
-## K8S
```

Comment:
Why removing the nuance between K8S and OpenShift? 

If the reader is on OpenShift, the quickstart might be possible in a different way and would also highlight some of the capabilities built around SBO.

Reply:
The installation part has some differences between OpenShift and Kubernetes. But installation is covered on another page. I think we can try to provide a uniform experience in the quickstart for both OpenShift and Kubernetes.